ridgeback_navigation/launch/gmapping_demo.launch

```diff
@@ -1,7 +1,10 @@
 <launch>
+  <arg name="scan_topic" default="front/scan" />
```

Comment:
Shouldn't this be <arg name="scan_topic" default="$(eval optenv('RIDGEBACK_LASER_TOPIC', 'front/scan'))" />?

Reply:
See above